src/ovh-angular-sidebar-menu.provider.js

```diff
@@ -504,18 +504,36 @@ angular.module("ovh-angular-sidebar-menu").provider("SidebarMenu", function () {
         /* ----------  HELPERS  ----------*/
 
         function getItemStateInfos (item) {
-            var infos = {
+            if (_.isString(item.loadOnState)) {
```

Comment:
Here, we duplicate the logic, I think is better to push `item.loadOnState` into an array, and always play the same logic based on fact that item.loadOnState is an array :)

What do you think ?

Something like:

```javascript
function getItemStateInfos (item) {
	let states;

	if(_.isString(item.loadOnState)) {
		states = [item.loadOnState];
	} else if (_.isArray(item.loadOnState) {
		states = item.loadOnState;
	} else {
		throw new Error('loadOnState attribute should be an array or a string');
		return;
	}

	// Let's do some stuff
}
```